dice-game-ui-angular/src/app/services/local-game-storage-service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Guid } from 'guid-typescript';
+import { GameEngine, SimpleGameEngine, Size } from 'engine';
+import { GameStorageService } from './game-storage-service';
+
+@Injectable()
+export class LocalGameStorageService implements GameStorageService {
+  createGame(gameEngine: GameEngine): Guid {
+    const guid = Guid.create();
+    localStorage.setItem(guid.toString(), gameEngine.fieldSize.width.toString());
+    return guid;
+  }
+
+  saveGame(gameEngine: GameEngine, guid: Guid): Guid {
+    if (!localStorage.getItem(guid.toString())) {
+      throw new Error('The saved game was not found in the store');
+    }
+    localStorage.setItem(guid.toString(), gameEngine.fieldSize.width.toString());
+    return guid;
+  }
+
+  restoreGame(guid: Guid): GameEngine {
+    const fieldSize: number = JSON.parse(localStorage.getItem(guid.toString()));
+    // temporary decision!!
```

Comment:
What exactly is `temporary decision`?

Reply:
All code from this line up to the end of the function. Again we are waiting for an internal way of the game engine class (of factory) to save itself and restore the object.

Reply:
Got it.

Reply:
Just removed my comment.